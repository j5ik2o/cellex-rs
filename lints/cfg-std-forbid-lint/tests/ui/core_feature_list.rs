#![deny(cfg_std_forbid)]

#[cfg(any(feature = "std", feature = "alloc"))]
fn mixed_guard() {}

fn main() {}
