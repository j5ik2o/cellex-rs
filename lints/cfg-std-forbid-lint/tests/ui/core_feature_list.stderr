error: `#[cfg(feature = "std")]` の使用が検出されました
  --> $DIR/core_feature_list.rs:3:1
   |
LL | #[cfg(any(feature = "std", feature = "alloc"))]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: std 依存コードは std 対応クレートへ移動するか、必要な範囲で `#![allow(cfg_std_forbid)]` を付与してください
   = note: AI向けアドバイス: std が不可欠なロジックは別モジュールへ切り離し、lint の適用境界を明確にしましょう。
note: the lint level is defined here
  --> $DIR/core_feature_list.rs:1:9
   |
LL | #![deny(cfg_std_forbid)]
   |         ^^^^^^^^^^^^^^

error: aborting due to 1 previous error

